test/Factory/AbstractMailFactoryTest.php

```diff
@@ -63,4 +63,32 @@ public function testCanCreateRequestedService(): void
         $result = $this->subject->canCreate($container, $requestedName);
         $this->assertTrue($result);
     }
+
+    /**
+     * @throws ContainerExceptionInterface
+     * @throws Exception
+     * @throws NotFoundExceptionInterface
+     */
+    public function testCanCreateWrongRequestedName(): void
```

Comment:
Isn't this identical to the next test? Both seem to use an invalid `$requestedName`.